### PR TITLE
Header titles added

### LINKED ISSUE #1611: Add headers to pages
Currently different pages don't have meta titles added.
We can add titles to -

when in home,
when inside folders,
when inside a shared folder,
when in bin,
when in search page,
when in settings.  

We can get inspiration from drive and dropbox.

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #1611

---
Submission checklist: 
No UI changes

### CODE REVIEW

**Comment by asnaith** [comment]
There wasn't too much for me to test here but I did run through the pages listed on the issue on both desktop / mobile and everything visually looked as expected :)

**Review by FSM1 — APPROVED** [review]

File: packages/files-ui/src/Components/Modules/FileBrowsers/SharedFileBrowser.tsx [hunk 1]
```diff
@@ -87,6 +88,9 @@ const SharedFileBrowser = () => {
       }
     })
   }), [arrayOfPaths, bucket, redirect])
+  const currentFolder = useMemo(() => {
+    return arrayOfPaths[arrayOfPaths.length - 1]
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/FileBrowsers/SharedFileBrowser.tsx]: should we go safe here and make sure this never end-up being `arrayOfPaths[-1]`?
```suggestion
    return !!arrayOfPaths.length && arrayOfPaths[arrayOfPaths.length - 1]
```

File: packages/files-ui/src/Components/Pages/BillingHistory.tsx [hunk 2]
```diff
@@ -34,20 +35,25 @@ const BillingHistory = () => {
...
-      </Typography>
-      <InvoiceLines payInvoice={(invoiceId) => setInvoiceToPay(invoiceId)} />
-      {invoiceToPay && <PayInvoiceModal
-        invoiceId={invoiceToPay}
-        onClose={() => setInvoiceToPay(undefined)}
-      />}
-    </div>
+    <>
+      <Helmet>
+        <title>Billing history - Chainsafe Files</title>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Pages/BillingHistory.tsx]: Translation
```suggestion
        <title><Trans>Billing history - Chainsafe Files</Trans></title>
```

File: packages/files-ui/src/Components/Pages/BinPage.tsx [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 import BinFileBrowser from "../Modules/FileBrowsers/BinFileBrowser"
+import { Helmet } from "react-helmet-async"
 
 const BinPage = () => {
   usePageTrack()
 
-  return <BinFileBrowser />
+  return <>
+    <Helmet>
+      <title>Bin - Chainsafe Files</title>
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Pages/BinPage.tsx]: Trans
```suggestion
      <title><Trans>Bin</Trans> - Chainsafe Files</title>
```

**Review by Tbaut — COMMENTED** [review]
We are missing translation for the title, I just commented for some of them, but we'd need them for all.

**Comment by tanmoyAtb** [comment]
> We are missing translation for the title, I just commented for some of them, but we'd need them for all.

Thanks for that, Added the translation tags, I left out "Chainsafe Files" part for translations as we've kept it that way everywhere else

**Review by Tbaut — CHANGES_REQUESTED** [review]
Code looks good but when I logged into Files, on the render preview, I got an error:
> Helmet expects a string as a child of <title>. Did you forget to wrap your children in braces? ( <title>{``}</title> ) Refer to our API for more information.

I guess the Trans wasn't expected there. so we probably need {`${t`title`} - Chainsafe Files`} or something along this line

**Comment by tanmoyAtb** [comment]
> Code looks good but when I logged into Files, on the render preview, I got an error:
> 
> > Helmet expects a string as a child of <title>. Did you forget to wrap your children in braces? ( <title>{``}</title> ) Refer to our API for more information.
> 
> I guess the Trans wasn't expected there. so we probably need {`${t`title`} - Chainsafe Files`} or something along this line

Thanks for saving this one, updates are in.

**Review by Tbaut — APPROVED** [review]
I tested Files and it looks good.

**Pull request merged**